index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <!-- Some meta tags -->
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portofolio Website</title>
+    <!-- Link to the Font-Awesome Font -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.2/css/font-awesome.min.css" />
+    <!-- Link to Roboto Font-->
+    <link href="http://fonts.googleapis.com/css?family=Roboto" rel="stylesheet" type="text/css" />
+    <!-- Styles -->
+    <!-- Custom Styles -->
+    <link rel="stylesheet" href="css/style.css" />
+</head>
+
+<body>
+    <nav class="nav-container">
+        <ul class="navbar-list">
+            <li class="name1">John Deo</li>
+            <li class="hamburger-menu"><i class="fa fa-bars"></i></li>
+        </ul>
+    </nav>
+    <header class="header-container">
+        <div class="content">
+            <div class="ellipse">
+                <img src="./images/Ellipse-1.svg" alt="" />
+            </div>
+            <div class="personal-description">
+                <div class="name2">
+                    <h1>Laura Jong</h1>
+                </div>
```

Comment:
- [ ] OPTIONAL: It would also be nice if you use your proper name instead of Laura Jong